_source/lib/apollos.coffee

```diff
@@ -81,41 +81,75 @@ Apollos.person = (user) ->
     person = Apollos.people.findOne
       guid: new RegExp(user.personGuid, "i")
 
-
-  # we shouldnt have any code for rock in the core package
+  # TODO we shouldnt have any code for rock in the core package
   # this will probably need to adjust the rock package
 
-  # if not person and user.rock and user.rock.personId
-  #   person = Apollos.people.findOne
-  #     personId: user.rock.personId
+  # PersonGuid would fall into the same category.  The only difference is that
```

Comment:
I would prefer to setup relationships in the core package. The system shouldn't rely on Rock, but rather it is a data source and utility for actions. When we begin to sync users from EE and other sources, we won't be able to rely on Rock having knowledge at creation.

Rock should be able to map its needed information using the Rock object on the person and the personGuid that Apollos generates. The Apollos.person.personGuid doesn't have to be the Rock personGuid